kolibri/plugins/learn/assets/src/composables/useLearningActivities.js

```diff
@@ -132,6 +132,38 @@ export default function useLearningActivities(contentNode) {
       : coreStrings.$tr('longActivity');
   });
 
+  /**
+   * @returns {String} Returns the translated exercise type(Practice Quiz or other)
+   */
+  const exerciseDescription = computed(() => {
+    const isPracticeQuiz = lodashGet(contentNode, ['options', 'modality'], false) === 'QUIZ';
+    const options_mastery_model = lodashGet(
```

Comment:
Don't need to read the mastery model from the options, only need the modality above. Reading this twice is liable to cause confusion - you can rely on the assessment metadata as canonical.

Apologies if my suggestion for using the 'options' confused this, but I meant only for the modality, not for anything else!

Reply:
@rtibbles Thank you for guiding me. I have removed options_mastery_model completely. I have another question, should I also make same changes to `ContentNodeThumbnail`, which is shown when 'i' is clicked for more information so that `Practice quiz` or `Get x correct` is seen in `ContentNodeThumbnail`.
![image](https://user-images.githubusercontent.com/110763795/233711366-66d8f902-7424-4d85-ac88-0f81582c64a8.png)

Reply:
I think this is fine for now - we can create a follow up issue for this is if needed.